review/tests/conftest.py

```diff
@@ -9,15 +9,15 @@
 
 @pytest.fixture(scope='function')
 def create_review() -> Callable[[str, str, str, str, int], Review]:
-    def _review_factory(client: str, professional: str, rating: str, description: str, days: int) -> Review:
+    def _review_factory(demo_client: str, professional: str, rating: str, description: str, days: int) -> Review:
```

Comment:
Please use the make_client fixture instead of client
https://github.com/redhat-beyond/Pro4U/blob/main/conftest.py#L114

You can override parameters if needed

same goes for make_professional fixture

Reply:
> Please use the make_client fixture instead of client https://github.com/redhat-beyond/Pro4U/blob/main/conftest.py#L114
> 
> You can override parameters if needed
> 
> same goes for make_professional fixture

Both fixtures still exist, should I change?

Reply:
Correct but @idosinger98 and @TalReinfeld 
Made some changes in the past, and they worked around the demo_client usage by others.
I think they had to delete it, 
Please open a ticket for removing the fixtures that do not use the factories

Reply:
> 

I will be using `demo_client`

Reply:
Please fix it, 
It is working,

Reply:
> Please fix it, It is working,

you mean use `make_client` instead of `demo_client` and `demo_client2`?

Reply:
Please avoid using demo_client and demo_client2
Those are deprecated methods

Reply:
> Please avoid using demo_client and demo_client2 Those are deprecated methods

Resolved